pkg/infra/game/state/view.go

```diff
@@ -22,17 +22,18 @@ type (
 	StaminaRange uint
 )
 
-const (
-	LowHealth  uint = 250 // 25% starting HP
-	MidHealth  uint = 500
-	HighHealth uint = 750 // 75% starting HP
-)
+var StartingHealthPoints = 1000
+var StartingStamina = 2000
 
-const (
-	LowStamina  uint = 500 // 25% starting HP
-	MidStamina  uint = 1000
-	HighStamina uint = 1500 // 75% starting HP
-)
+// var StartingHealthPoints, _ = strconv.ParseInt(os.Getenv("STARTING_HP"), 10, 0)
+// var StartingStamina, _ = strconv.ParseInt(os.Getenv("BASE_STAMINA"), 10, 0)
+
+var granularity = 10
+
+// var granularity, _ = strconv.ParseInt(os.Getenv("GRAN"), 10, 0)
+
+var HealthQuant = int(StartingHealthPoints) / granularity
+var StaminaQuant = int(StartingStamina) / granularity
```

Comment:
Cannot seem to be able to read the environment variables from `.env`... For some reason the commented out code always returns 0 as the value

Reply:
yeah thats strange. But considering we don't change the health. For the purposes of the paper, this method is fine.